Confluent.Kafka.sln

```diff
@@ -1,4 +1,4 @@
-Microsoft Visual Studio Solution File, Format Version 12.00
+﻿Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
MS tools are great at creating pointless formatting diffs between versions and whatnot,
can't even open librdkafka in VS without it messing with the sln file. 🇦🇷